src/frontend/src/pages/HomePage/components/Task.tsx

```diff
@@ -3,19 +3,47 @@ import { useState } from 'react';
 import { KeyboardArrowRight, KeyboardArrowDown } from '@mui/icons-material';
 import SubtaskSection from './SubtaskSection';
 import { Checklist } from 'shared';
+import { useCheckedChecklists, useToggleChecklist } from '../../../hooks/onboarding.hook';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { useToast } from '../../../hooks/toasts.hooks';
 
 interface SubtaskProps {
   subtasks: Checklist[];
```

Comment:
I know this is from a previous ticket but is there a reason to have this prop? I think I would prefer to prop drill just the parent checklist and pull out the children, but I'm open to hear your thoughts